executor/join.go

```diff
@@ -450,6 +576,31 @@ func (e *HashJoinExec) join2Chunk(workerID uint, probeSideChk *chunk.Chunk, hCtx
 	}
 	return true, joinResult
 }
+func (e *HashJoinExec) join2ChunkForOuterHashJoin(workerID uint, probeSideChk *chunk.Chunk, hCtx *hashContext, joinResult *hashjoinWorkerResult) (ok bool, _ *hashjoinWorkerResult) {
```

Comment:
Add a comment for this function. The reader may not know what's OuterHashJoin

Reply:
ok